monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -16,16 +34,34 @@ class SCDTestIssue(ImplicitDict):
     details: str
     '''Human-readable description of the issue'''
 
-    def __init__(self, **kwargs):
-        super(SCDTestIssue, self).__init__(**kwargs)
-        if 'timestamp' not in kwargs:
-            self.timestamp = datetime.datetime.utcnow().isoformat()
+    queries: List[fetch.Query]
+    """Description of HTTP requests relevant to this issue"""
 
 class RequiredResult(ImplicitDict):
     ''' A class to evaluate results / response to an injection of test flight data (TestFlightRequest) '''
-    expected_result: str
-    incorrect_result_details: Optional[SCDTestIssue]
+    acceptable_results: List[str] # Holds information about the "result" response provided by the test interface of the USS under test. Per the SCD Testing API this should be one of "Planned", "Rejected", "ConflictWithFlight" or "Failed"
+    incorrect_result_details: Optional[Dict[Result, KnownIssueFields]] # In the case where the USS under test provides a result that is not the same as expected result then the test driver can provide a message detailing why the expected result was populated the way it was. This should help the USS under test debug specific parts of their internal systems. 
+
+class InjectionTarget(ImplicitDict):
+    ''' A class to hold details of a USS under test '''
+    name: str
+    uss_base_url: str
 
-class TestInjectionRequiredResult(ImplicitDict):
+class SCDTestConfiguration(ImplicitDict):
+    ''' A class to hold test configuration, injection target data  '''
+    injection_targets: List[InjectionTarget]
+
+class FlightInjectionAttempt(ImplicitDict):
+    ''' A class to hold details of the test injection and the expected result of processing the flight request '''
     test_injection: InjectFlightRequest
-    required_result: RequiredResult
\ No newline at end of file
+    required_result: RequiredResult
+
+class TestDescription(ImplicitDict):
+    ''' A class to hold different test structures '''
+    name: str # e.g. a Nominal test or Flight Authorisation test
+    response_wait_time_secs: int # amount of time the test driver should wait before expecting a response
```

Comment:
Why is a behavior part of a class labeled "Description"?

Reply:
I have now renamed the Description -> Details to accommodate the behaviour